lib/pages/new_group/new_group.dart

```diff
@@ -55,6 +55,13 @@ class NewGroupController extends State<NewGroup>
     fetchContactsController.fetchCurrentTomContacts();
     listenForGroupchatInfoScrollController();
     fetchContactsController.listenForScrollChanged(fetchContactsController: fetchContactsController);
+    searchContactsController.onSearchKeywordChanged = (searchKey) {
+      disableLoadMoreInSearch();
+    };
+  }
+
+  void disableLoadMoreInSearch() {
```

Comment:
IMO this names method is not relevant it should be `checkLoadMoreInSearch` since you disable if search is empty